Exercises/Excercise14/solution14.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Objects;
+
+class ComplexNumber{
+    float real;
+    float imaginary;
+    ComplexNumber(float real,float imaginary){
+        this.real = real;
+        this.imaginary = imaginary;
+    }
+    @Override
+    public String toString() {
+
+        return "ComplexNumber{" +
+                "real=" + real +
+                ", imaginary=" + imaginary +"i }";
+    }
+}
+abstract class Arithmetics{
+    final float add(float f,float g){
+        return f+g;
+    }
+    final float sub(float v,float j){
+        return v-j;
+    }
+    final float mul(float m,float n){
+        return m*n;
+    }
+    abstract void display(ComplexNumber c);
+
+
+    static class ComplexArith extends Arithmetics
+    {
+        ComplexNumber complexAdd(ComplexNumber c,ComplexNumber f){
+            System.out.println();
+            System.out.println("Addition result:");
+            return new ComplexNumber (super.add(c.real,f.real),super.add(c.imaginary, f.imaginary));
+        }
+        ComplexNumber complexSub(ComplexNumber c,ComplexNumber f){
+            System.out.println();
+            System.out.println("Subtraction result:");
+            return new ComplexNumber (super.sub(c.real,f.real),super.sub(c.imaginary, f.imaginary));
+        }
+        ComplexNumber complexMul(ComplexNumber c,ComplexNumber f){
+            System.out.println();
+            System.out.println("Division result:");
+            return new ComplexNumber (super.mul(c.real,f.real),super.mul(c.imaginary, f.imaginary));
```

Comment:
Check the logic for complex number multiplication

Reply:
Modified the logic for complex multiplication:
Given complex numbers:
15.8  18.9i
9.8  6.9i

Addition result:
ComplexNumber{real=25.6, imaginary=25.8i }

Subtraction result:
ComplexNumber{real=6.0, imaginary=12.0i }

Multiplication result:
ComplexNumber{real=24.430008, imaginary=294.24i }